app/controllers/actor_movies_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ActorMoviesController < ApplicationController
+  def create
+    actor_movie = ActorMovie.create!(actor_id: params[:actor_id], movie_id: params[:id])
+    movie = Movie.find_by(id: actor_movie.movie_id)
```

Comment:
I believe you could also just use the params here rather than using a `find_by` query to set the movie:
```ruby
movie = Movie.find_by(id: params[:id])
```